state.json

```diff
@@ -0,0 +1 @@
+{"validators":[{"peer_id":"0x0061a01516c66e924069c07aa657b311e6404390659a376d41e2ea9ae5abc60ac375cbbe4d5098f560085185cdfbb540545b8f9eca1c357e79de7f3614d249a6","voting_power":100,"priority":100.0},{"peer_id":"0xf51e06e0a3ec07cbb16dd41e27a4d30a3d137612aa1c3e846fb1283076449ccbae8e7239cdcdc4e7e88345b5b472d26232ac1bfd220dd881eb7bb7f71974d3de","voting_power":100,"priority":-100.0}],"new_joiner_penalty_factor":1.125,"block_number":5,"last_proposer":"0xf51e06e0a3ec07cbb16dd41e27a4d30a3d137612aa1c3e846fb1283076449ccbae8e7239cdcdc4e7e88345b5b472d26232ac1bfd220dd881eb7bb7f71974d3de"}
```

Comment:
this is legacy and should be deleted. If its still there when test running. Means its due to a failed merge attempt in the leader selection algo